html/includes/reports/alert-log.pdf.inc.php

```diff
@@ -41,24 +43,24 @@
     $hostname    = gethostbyid(mres($alert_entry['device_id']));
     $alert_state = $alert_entry['state'];
 
-    if ($alert_state != '') {
-        if ($alert_state == '0') {
+    if ($alert_state != '') { // FIXME: would is_numeric($alert_state) be more appropriate here?
```

Comment:
@f0o: what do you think on this one?  Could $alert_entry['state'] be null?  If so, should it be treated the same as 0?